apis/r/R/utils.R

```diff
@@ -165,13 +165,13 @@ uns_hint <- function(type = c('1d', '2d')) {
   }
   x$reopen("READ", tiledb_timestamp = x$tiledb_timestamp)
   dimname <- x$dimnames()[axis + 1L]
-  rl <- sr_setup(
+  sr <- sr_setup(
     uri = x$uri,
-    config = as.character(tiledb::config(x$tiledbsoma_ctx$context())),
+    soma_context(),
```

Comment:
Note for future work: this should be replaced with fetching the context from `x`; currently available with
```suggestion
    x$.__enclos_env__$private$.soma_context
```

Reply:
Fair point but not urgent and on the fence as whether we should use opaque internals.  "Future work", as you say.